armi/operators/operator.py

```diff
@@ -393,7 +394,15 @@ def _performTightCoupling(self, cycle: int, timeNode: int, writeDB: bool = True)
                 self.r.core.p.coupledIteration = coupledIteration + 1
                 converged = self.interactAllCoupled(coupledIteration)
                 if converged:
+                    runLog.extra(
```

Comment:
this should be at least info and my suggestion is "important".

Reply:
Addressed. https://github.com/terrapower/armi/pull/1172/commits/7b7cbf86deb5494b16cb2ac7e6b6d9e1fe449df2